src/test/java/apprenticeship/BoardTest.java

```diff
@@ -11,10 +11,9 @@
 
     @Before
     public void initBoard() {
-        tictactoe = new Board();
+        tictactoe = new Board(9);
     }
```

Comment:
Since you are going to be using `Board(9)` in most cases, maybe the initializer should default to that option?